molo/core/tests/test_views.py

```diff
@@ -351,6 +351,39 @@ def test_featured_homepage_listing_in_french(self):
             response,
             '<p>Sample page description for 0 in french</p>')
 
+    def test_page_moving(self):
+        self.yourmind_fr = self.mk_section_translation(
+            self.yourmind, self.french, title='Your mind in french')
+        self.yourmind_ar = self.mk_section_translation(
+            self.yourmind, self.arabic, title='Your mind in arabic')
+        en_page = self.mk_article(self.yourmind)
+
+        response = self.client.get('/')
+        self.assertContains(
+            response, '<a href="/sections/your-mind/">Your mind</a>')
+        response = self.client.get('/sections/your-mind/%s/' % (en_page.slug))
+        self.assertContains(
+            response, ' <p>Sample page content for 0</p>')
+        fr_page = self.mk_article_translation(
+            en_page, self.french,
+            title=en_page.title + ' in french',
+            subtitle=en_page.subtitle + ' in french',
+            body=json.dumps([{
+                            'type': 'paragraph',
+                            'value': 'Sample page content for %s' % (
+                                en_page.title + ' in french')}]),
+        )
+
+        response = self.client.get('/locale/fr/')
+
+        response = self.client.get('/sections/your-mind/%s/' % (fr_page.slug))
+        self.assertContains(response, 'Sample page content for %s' % (
+            en_page.title + ' in french'))
+
+        en_page.move(self.yourmind_sub, pos='last-child')
+        self.assertEquals(en_page.get_parent(), self.yourmind)
+        self.assertEquals(fr_page.get_parent(), self.yourmind)
```

Comment:
@Ntu2koTrevor both `self.yourmind` should be `self.yourmind_sub` as you are moving the `en_page` from `yourmind`section to `yourmind_sub` section